src/pages/home.js

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getData } from '../Redux/Reducers/data';
+import southAmerica from '../images/Southamerica.svg';
+
+const Home = () => {
+  const sac = [
+    'Argentina',
+    'Bolivia',
+    'Brazil',
+    'Chile',
+    'Colombia',
+    'Ecuador',
+    'Guyana',
+    'Paraguay',
+    'Peru',
+    'Suriname',
+    'Uruguay',
+    'Venezuela',
+  ];
+
+  const dispatch = useDispatch();
+  const data = useSelector((state) => state.data.filter((country) => sac.includes(country.name)));
+
+  useEffect(() => {
+    dispatch(getData());
+  }, []);
+
+  const handleRedirect = (country) => {
+    window.location.href = `/${country}`;
+  };
+
+  const handleAbout = () => {
+    window.location.href = '/about';
+  };
+
+  return (
+    <div className="Home">
+      <h1 className="title">Covid 19 in South American Countries</h1>
+      <div className="first-card-cont">
+        <img className="sac-img" src={southAmerica} alt="South american countries" />
+        <div className="first-card">
+          <h2 className="subtitle">South America</h2>
+          <p className="description">442.5M hab</p>
+          <button type="button" className="btn-about" onClick={handleAbout}>About</button>
+        </div>
+      </div>
+      <div className="second-card-cont">
+        <h2 className="stats">STATS BY COUNTRY</h2>
+        <div className="second-card">
```

Comment:
- [ ] Your project is complete, just one thing left is to add a filter by the country as mentioned on the requirements `one page with a list of items that could be filtered by some parameters; in the example below, it's a list of metrics that can be filtered by the country (imagine a search field to introduce the country name like Italy, Croatia, etc.). This page should be your homepage.`

![image](https://user-images.githubusercontent.com/35535130/168946481-6b123b3f-b3f8-41e7-8fac-de323b3fda1c.png)

Reply:
Thank you its done now